src/esw/brushed_motors/Controller.cpp

```diff
@@ -101,21 +101,22 @@ void Controller::moveOpenLoop(float input) {
 }
 
 // REQUIRES: nothing
-// MODIFIES: isCalibrated
-// EFFECTS: asks the MCU if it is calibrated
-void Controller::askIsCalibrated() {
+// MODIFIES: nothing
+// EFFECTS: returns if the MCU is calibrated
+bool Controller::isCalibrated() {
```

Comment:
For any function that uses the I2C bus, it may be worth adding that it uses I2C in the EFFECTS section of the docstring thing